### PR TITLE
Change URLs with mila-udem to epistimio

### PR DESCRIPTION
**Pull request by bouthilx**
Why:

The repository of Oríon has been moved to Epistímio organization.

### CODE REVIEW

File: src/orion/core/__init__.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     'tsirif': ('Christos Tsirigotis', 'tsirif@gmail.com'),
     'bouthilx': ('Xavier Bouthillier', 'xavier.bouthillier@gmail.com'),
 }
-__url__ = 'https://github.com/mila-udem/orion'
```
Comment (tsirif) [inline src/orion/core/__init__.py]: (I am not sure why I cannot comment on something else apart from the changes, but I think what I am about to write is relevant here in this file and PR)

We should probably apply corresponding changes to the private-magic variables above this one, with respect to the `MILA` -> `Epistimio` change.

Comment (bouthilx) [reply src/orion/core/__init__.py]: Indeed. I did it in Kleio but forgot here. I'll add this and rebase.

Comment (bouthilx) [reply src/orion/core/__init__.py]: Done

Comment (bouthilx) [reply src/orion/core/__init__.py]: @tsirif Ping :)

Comment (nouiz) [reply src/orion/core/__init__.py]: @tsirif Ping :)

**Comment by bouthilx** [comment]
PR rebased, waiting for approval.

File: src/orion/core/__init__.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 __version__ = VERSIONS['version']
 __license__ = 'BSD-3-Clause'
 __author__ = u'Epistímio Team'
+__author_short__ = u'Epistímio Team'
```
Comment (tsirif) [inline src/orion/core/__init__.py]: @bouthilx what is the need for this? Shall we name it without the ` Team` in front?

Comment (bouthilx) [reply src/orion/core/__init__.py]: I named it `Epistímio Team` because it was `Oríon Team`, but I do prefer `Epistímio` indeed.

Comment (DatCorno) [reply src/orion/core/__init__.py]: Same here, I preter `Epistímio`

**Pull request merged**